webapp/services/add_user.py

```diff
@@ -26,7 +29,7 @@ def post(self):
 				name = user_name, 
 				payment_key = user_payment_key,
 				email = user_email,
-				date_of_birth = user_date_of_birth
+				date_of_birth = datetime.strptime(user_date_of_birth, '%Y-%m-%d %H:%M:%S')
```

Comment:
Pretty sure this is going to break the Android app. I'll fix it once this is in.

Reply:
ok.. i tested sending something like 
{ userName = "Rayo Landeros", 
userPaymentKey = "st_247892895902",
userEmail = "rayolanderos@gmail.com",
userDateOfBirth = "1987-06-02 00:00:00" }